packages/contracts/test/adaptive-erc165.ts

```diff
@@ -102,4 +102,4 @@ describe('AdaptiveErc165', function () {
       expect(await adaptive.supportsInterface('0xabababab')).to.equal(false);
     });
   });
-});
+});
```

Comment:
No newline at end of file

Reply:
My git ignores the whitespace, even with prettier applied. Btw, let's use GitHub suggestions where applicable.
```suggestion
});

```